### PR TITLE
Translations(Spanish):5-clustering/readme.md

### PR DESCRIPTION
**Pull request by hereje**
Translate file `5-Clustering/README.md'` to spanish

### CODE REVIEW

**Comment by jlooper** [comment]
Thank you!

**Comment by Javieiraa** [comment]
Please can someone help me understand this GitHub stuff

File: 5-Clustering/translations/README.es.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Lecciones
+
+1. [Introducción al clustering](1-Visualize/README.md)
+2. [K-Means clustering](2-K-Means/README.md)
+
+## Créditos
+
+Éstas lecciones fueron escritas con 🎶 por [Jen Looper](https://www.twitter.com/jenlooper) con la útil ayuda de [Rishit Dagli](https://rishit_dagli) y [Muhammad Sakib Khan Inan](https://twitter.com/Sakibinan).
+
+El conjunto de datos de las [Canciones Nigerianas](https://www.kaggle.com/sootersaalu/nigerian-songs-spotify) fue obtenido de Kaggle como raspado de Spotify.
```
Comment (alfredodeza) [inline 5-Clustering/translations/README.es.md:27]: I'm not sure what _raspado_ means here

File: 5-Clustering/translations/README.es.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+![Un tornamesa](../images/turntable.jpg)
+
+> Foto por <a href="https://unsplash.com/@marcelalaskoski?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Marcela Laskoski</a> en <a href="https://unsplash.com/s/photos/nigerian-music?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+  
+En esta serie de lecciones, descubrirás nuevas formas de analizar datos usando técnicas de clustering. El clustering es particularmente útil cuando tu conjunto de datos carece de etiquetas. Si este sí tiene etiquetas, entonces las técnicas de clasificación como las que has aprendido en lecciones previas son más útiles. Pero en casos donde pretendes agrupar datos sin etiquetas, el clusterin es una gran forma de descubrir patrones.
```
Comment (alfredodeza) [inline 5-Clustering/translations/README.es.md:13]:
```suggestion
En esta serie de lecciones, descubrirás nuevas formas de analizar datos usando técnicas de clustering. El clustering es particularmente útil cuando tu conjunto de datos carece de etiquetas. Si este sí tiene etiquetas, entonces las técnicas de clasificación como las que has aprendido en lecciones previas son más útiles. Pero en casos donde pretendes agrupar datos sin etiquetas, el clustering es una gran forma de descubrir patrones.
```

**Review by alfredodeza — CHANGES_REQUESTED** [review]
You seem to also have a _merge_ in your commits. Can you clean that up? I tend to rebase and fix the merge commit that way, but feel free to try something else as well as long as that merge commit gets cleared

File: 5-Clustering/translations/README.es.md (hunk 1 above)
Comment (hereje) [reply 5-Clustering/translations/README.es.md]: It refers to data scraping; so I changed the expression from 'raspado de Spotify' to 'raspado de datos de Spotify'

**Comment by jlooper** [comment]
how are we doing on this?

**Review by jlooper — APPROVED** [review]
looks clean to me so merging!

**Pull request merged**